### DISCUSSION TITLE
GitHub Support does not respond even after 7 days #2718458

### DISCUSSION THREAD

**Discussion by Lakshmiichoudhary** [community/community#119643] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear Github Support Team and community members,

Please restore my teammate account. His account is flag and not visible for other user. He is waiting more than a week but still not getting response from GitHub team🥹🥹🥹.

Tickets: #2718458
Username: Junaid1240651

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Lakshmiichoudhary ! The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

 The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
